src/components/nodes/messageFlow/messageFlow.vue

```diff
@@ -62,7 +71,7 @@ export default {
         return true;
       }
 
-      if (this.targetIsIntermediateCatchEvent()) {
+      if (this.targetIsIntermediateCatchEvent() || this.targetTaskTypes()) {
```

Comment:
If you implement the comment above this can be change to:
```suggestion
      if (this.isValidTarget(this.targetNode.definition.$type)) {
```